content/docs/command-reference/repro.md

```diff
@@ -184,17 +182,15 @@ best
 And runs a few simple transformations to filter and count numbers:
 
 ```dvc
-$ dvc run -f filter.dvc -d text.txt -o numbers.txt \
+$ dvc run -n filter -d text.txt -o numbers.txt \
            "cat text.txt | egrep '[0-9]+' > numbers.txt"
 
-$ dvc run -f Dvcfile -d numbers.txt -d process.py -M count.txt \
+$ dvc run -n count -d numbers.txt -d process.py -M count.txt \
            "python process.py numbers.txt > count.txt"
 ```
 
-> Note that using `-f Dvcfile` with `dvc run` above is optional, the stage file
-> name would otherwise default to `count.txt.dvc`. We use `Dvcfile` in this
-> example because that's the default stage file name `dvc repro` will read
-> without having to provide any `targets`.
+> Note that a stage name is required when executing `dvc run`. It can be
+> specified with `-n` (`--name`) option as we did above.
 
```

Comment:
Thanks! In this case I would just get rid of the note altogether.

Reply:
@sarthakforwet this is not updated, yet you resolved the conversation. Again please don't resolve conversations, I'll do that when I check again. Unresolving. Thanks